src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -824,9 +830,9 @@ export default class ConvasController {
         cc.save();
 
         if (!aircraft.inside_ctr) {
-            cc.fillStyle = this.theme.RADAR_TARGET_OUTSIDE_RANGE;
+            cc.fillStyle = this.theme.HISTORY_DOT_OUTSIDE_RANGE;
```

Comment:
I'm trying to show some restraint in not mucking with everything I see on this PR. That said, I kept an eye out for this sort of thing and made a few small changes with things like this. But you'll see I left some so I could be sure to finish in a timely manner.